src/core/ui/context/login-methods.tsx

```diff
@@ -88,11 +88,6 @@ const useLoginMethods = () => {
   const user = useUser();
 
   const web3 = useMemo(() => {
-    if (!viewOnly && user?.loginMethods.includes(LoginMethodType.Web3)) {
-      // User is already logged in using web3 so we should not expose this.
-      return [];
-    }
-
```

Comment:
@nedrocks When a user logged with a wallet, removes the wallet and tries to add a new one, no connection option is displayed in the sign-in modal. 
Why is this condition added here? Have you got a different approach in mind? @nedrocks

Reply:
viewOnly is used to support the styling screen. It's important to ensure people can't interact with the modal when they're styling it.